### ISSUE TITLE
feat: better formatting options for successful installed message?

### ISSUE THREAD

**Issue by notatallshaw** [prefix-dev/rip#175]
**Environment:** Linux Python 3.11

**Command**: `cargo r -- apache-airflow[all]==2.8.1 apache-beam==2.53.0`

**Output**: https://gist.github.com/notatallshaw/cd59776c2d78843ef08a24f11f96e495 or wrapped: 

**Request**:  This output message is unreadable on the terminal as the lines wrap around. 

For me it would be nice if the output was something simpler like `{package-requirement}=={installed-version}`. But it's a tough problem.

**Comment by tdejager** [comment]
This is happening because of the long list of `extra`'s you mean right :)?

**Comment by notatallshaw** [comment]
> This is happening because of the long list of `extra`'s you mean right :)?

Yeah, the extras list in this exampls is over 1000 characters long.

It's a pretty weird edge case, so certainly not a priority. But when I first encountered it I had no idea what was going on and it just looked like rip output was broken.

**Comment by tdejager** [comment]
Okay lets keep track of it at least for now :)
